OxfordBodleian/Juel-Jensen/BDLaethb4.xml

```diff
@@ -183,63 +1266,204 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                              - on other folia partly preserved: ff. 139v, 147r
+                          
+                           B
+                           - drawing of a horse and a figure in pencil on f. iv
+                           - drawing of two figures (a monk and a saint?) in ink on f. 3r, visible also on f. 3v      
+                           - a face in the margin on f. 119r, probably in imprint from the facing page with the miniature
                         </decoDesc>-->
-                        
-                     <!--   <bindingDesc>
-                            <binding xml:id="binding">
-                                <decoNote xml:id="b1"></decoNote>
-                                <decoNote xml:id="b2" type="bindingMaterial">
-                           <material key="wood"/>
-                        </decoNote>
+
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <locus target="#4r"/>
+                                    <desc type="OwnershipNote">Note written in black ink in the
+                                        upper margin, stating that this manuscript belonged to
+                                            <placeName ref="INS0101MadhaneAlam"/>.</desc>
+                                    <q xml:lang="gez">ገድለ፡ ጊዮርጊስ፡ ዘቅዱስ፡ <placeName
+                                            ref="INS0101MadhaneAlam">መድኃኔ፡ ዓለም፡</placeName></q>
+                                </item>
+
+                                <item xml:id="e1">
+                                    <desc>Scribal correction by the main hand are rare. The scribe
+                                        rarely added a missing word interlineally (e.g. <locus
+                                            target="#136rc"/>), he marked a dittographical error
+                                        with thin strokes above and below the words on <locus
+                                            target="#106rb"/> (in the same way as the numerals are
+                                        marked). Other corrections were most likely made by later
+                                        hands. Superfluous words were washed out carelessly at the
+                                        beginning of the codex (e.g. on <locus
+                                            target="#5vc #6ra #6vc"/>). Missing characters and words
+                                        were interlineally inserted, partly with a thin nib (e.g.
+                                            <locus target="#51va #110vc #146rb"/>). Words or
+                                        passages erroneously written twice (dittographies) were
+                                        encircled with a thin line (e.g. <locus
+                                            target="#40vc #133va #134ra"/>).</desc>
```

Comment:
Include all info on dittography here?

Dittographical errors are marked with a thin stroke above and below the words on <locus target="#106rb"/> (in the same way as the numerals are marked) or by encircling the errors with a thin line (e.g. etc.."

Reply:
I distinguish in this manuscript (because it is very obvious here) between the corrections made by the main hand and those made by later hands. Therefore, I would not merge the information on dittographies